### PR TITLE
games-engines/instead: new package

### PR DESCRIPTION
**Pull request by Fat-Zer**
Closes: https://bugs.gentoo.org/285086
Signed-off-by: Alexander Golubev <fatzer2@gmail.com>

### CODE REVIEW

File: games-engines/instead/instead-3.3.3.ebuild [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+if [[ ${PV} == *9999* ]]; then
+	EGIT_REPO_URI="https://github.com/instead-hub/${PN}.git"
+	inherit git-r3
+else
+	SRC_URI="https://github.com/instead-hub/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
+fi
+
+DESCRIPTION="INSTEAD text-based quest game engine"
+HOMEPAGE="http://instead3.syscall.ru/"
```
Comment (vaukai) [inline games-engines/instead/instead-3.3.3.ebuild:21]: Please try https for HOMEPAGE

Comment (Fat-Zer) [reply games-engines/instead/instead-3.3.3.ebuild]: fixed for both `3.3.3`&`9999`, amended and force-pushed

**Comment by Fat-Zer** [comment]
Update:

* Version bump to 3.3.4
* Update homepage URL

**Comment by Fat-Zer** [comment]
Manifest fixed.

**Comment by Fat-Zer** [comment]
The package won't likely be accepted to the main tree, so I've added it to the guru overlay instead (no pun intended).

**Pull request closed without merging**